<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<comment> Copyright (C) 2011 - 2015 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and The University </comment>
<comment> of Manchester. </comment>
<comment> All rights reserved. </comment>

 <class>CQMathMatrixWidget</class>
 <widget class="QWidget" name="CQMathMatrixWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="mpTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="CQArrayAnnotationsWidget" name="mpArrayWidget1">
      <attribute name="title">
       <string>Stoichiometry Matrix</string>
      </attribute>
     </widget>
     <widget class="CQArrayAnnotationsWidget" name="mpArrayWidget2">
      <attribute name="title">
       <string>Reduced Stoichiometry Matrix</string>
      </attribute>
     </widget>
     <widget class="CQArrayAnnotationsWidget" name="mpArrayWidget3">
      <attribute name="title">
       <string>Link Matrix</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string notr="true">Deriv test</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="mpDerivButton">
         <property name="text">
          <string>Deriv</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QtMmlWidget" name="mpMML" native="true"/>
       </item>
       <item row="2" column="0">
        <widget class="QtMmlWidget" name="mpMML2" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CQArrayAnnotationsWidget</class>
   <extends>QWidget</extends>
   <header>copasi/UI/CQArrayAnnotationsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>QtMmlWidget</class>
   <extends>QWidget</extends>
   <header location="global">qtmmlwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
